packages/Menu/src/Menu.tsx

```diff
@@ -54,8 +54,9 @@ function Menu({
   refEl,
   trigger,
   ...rest
-}: MenuProps): React.ReactElement {
+}: MenuProps) {
   const [isActive, setActiveState] = useState(false);
+  const EscapeKey = 27;
```

Comment:
I'd put it outside the component entirely.

Reply:
👍

Reply:
👍